File: src/app/models/car.ts
export class Car {
  id: number;
  plate_id: number;
  model: number;
  four_on_four: boolean;
  engine_size: number;
  year: number;
  comment: string;
  delivered: number;
  fix_date: Date;
  last_updated: Date;

  Car() {
    this.id = null;
    this.plate_id = null;
    this.model = null;
    this.four_on_four = null;
    this.engine_size = null;
    this.year = null;
    this.comment = null;
    this.delivered = null;
    this.fix_date = null;
    this.last_updated = null;
  }
}

File: webAPI/webAPI/Models/Car.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace webAPI.Models
{
    public class Car
    {
        [Key]
        public int id { get; set; }
        [Required]
        public int plate_id { get; set; }
        [Required]
        public int model { get; set; }
        [Required]
        public bool four_on_four { get; set; }
        [Required]
        public int engine_size { get; set; }
        [Required]
        public int year { get; set; }
        
        [Column(TypeName = "varchar(100)")]
        public String comment { get; set; }
        
        public int delivered { get; set; }
        [Required]
        public DateTime fix_date { get; set; }
        [Required]
        public DateTime last_updated { get; set; }


    }
}
